### DISCUSSION TITLE
Really needed?

### DISCUSSION THREAD

**Discussion by deleonio** [windicss/vite-plugin-windicss#102] [General]
![image](https://user-images.githubusercontent.com/6279703/111948058-d6707880-8ade-11eb-90de-65fa1fda38a2.png)

**Is it possible to live without?!**

**Comment by antfu** [comment]
Yeah, we can do that, but I feel it's better to explicitly import it so you have better controls of the order of the css injection. What's the problem you are running into with it?

**Comment by IRediTOTO** [comment]
this is not a problem. Let's look into your framework like NextJS they have an _app.js apply to all file, put it there.
